sdk/communication/azure-communication-common/build.gradle

```diff
@@ -1,8 +1,6 @@
 ext.publishName = "Microsoft Azure Android Client Library common code For Communication Service"
-description = "This package contains the Android client library common code for Microsoft Azure Communication Service.\n" +
-        "    For this release, see notes - https://github.com/Azure/azure-sdk-for-android/blob/master/sdk/communication/azure-communication-common/README.md and https://github.com/Azure/azure-sdk-for-android/blob/master/sdk/communication/azure-communication-common/CHANGELOG.md."
```

Comment:
Please restore these line since this will provide links to documentation in Maven